tests/test_transforms/test_impute_transform.py

```diff
@@ -50,12 +50,11 @@ def test_wrong_init_one_segment():
         _ = _OneSegmentTimeSeriesImputerTransform(strategy="wrong_strategy")
 
 
-# TODO: fix the test after changing behaviour of PerSegmentWrapper to fail in __init__
 @pytest.mark.xfail
```

Comment:
This line should be deleted, it shows that this test may fail and pytest ignores the result.

Reply:
Done